src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -88,11 +90,26 @@ void initializeReads() {
         if (hasReference()){
             // pass in reference if available, because CRAM files need it
             factory = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.SILENT).referenceSequence(referenceArguments.getReferenceFile());
+        } else if (hasCramInput()) {
+            throw new UserException("A reference file is required when using CRAM files.");
```

Comment:
Can you create a new subclass of `UserException` (within the existing `UserException.java` file) called `MissingReference`, throw a `UserException.MissingReference` here, and check that `UserException.MissingReference` specifically is thrown in your tests below? By checking for a more specific exception type, we can be more certain that the right error is being thrown.